p4proto/p4src/linux_networking/lnw.conf

```diff
@@ -0,0 +1,55 @@
+{
+    "chip_list": [
+        {
+            "chip_family": "dpdk",
+            "instance": 0,
+            "pcie_sysfs_prefix": "/sys/devices/pci0000:00/0000:00:03.0/0000:05:00.0",
+            "pcie_domain": 0,
+            "pcie_bus": 5,
+            "pcie_fn": 0,
+            "pcie_dev": 0,
+            "pcie_int_mode": 1,
+            "sds_fw_path": "share/tofino_sds_fw/avago/firmware"
+
+        }
+    ],
+    "instance": 0,
+    "p4_devices": [
+        {
+            "device-id": 0,
+            "eal-args": "dummy -n 4 -c 3",
+            "mempools": [
+                {
+                    "name": "MEMPOOL0",
+                    "buffer_size": 2304,
+                    "pool_size": 1024,
```

Comment:
this conf file looks to be old, please check the latest one.
We have modified this pool_size to 4096